File: backend/src/main/java/github/lms/lemuel/xr/common/jwt/JwtProperties.java
package github.lms.lemuel.xr.common.jwt;

import org.springframework.boot.context.properties.ConfigurationProperties;

/** application.yml 의 security.jwt.*. */
@ConfigurationProperties(prefix = "security.jwt")
public record JwtProperties(
        String secret,
        int expiryDays,
        String issuer
) {}

File: backend/src/main/kotlin/github/lms/lemuel/xr/common/jwt/JwtProperties.kt
package github.lms.lemuel.xr.common.jwt

import org.springframework.boot.context.properties.ConfigurationProperties

/** application.yml 의 security.jwt.*. */
@ConfigurationProperties(prefix = "security.jwt")
data class JwtProperties(
    val secret: String,
    val expiryDays: Int,
    val issuer: String,
)
